packages/vue-components/src/panels/PanelBase.js

```diff
@@ -80,12 +84,18 @@ export default {
     hasHeaderBool() {
       return this.$slots.header;
     },
+    expandHeaderlessBool() {
+      return toBoolean(this.expandHeaderless);
```

Comment:
the additional `toBoolean` shouldn't be required since you're using `type: Boolean` above

Reply:
Okay, I will make the changes